apps/lcs/behaviours/send-email-notification.js

```diff
@@ -0,0 +1,139 @@
+const config = require('../../../config');
+const NotifyClient = require('notifications-node-client').NotifyClient;
+const notifyKey = config.govukNotify.notifyApiKey;
+const translation = require('../translations/src/en/fields.json');
+const notifyClient = new NotifyClient(notifyKey);
+const moment = require('moment');
+const PRETTY_DATE_FORMAT = 'DD MMMM YYYY';
+
+const getLabel = (fieldKey, fieldValue) => {
+  if (Array.isArray(fieldValue)) {
+    return fieldValue.map(option => translation[fieldKey].options[option].label).join(', ');
+  }
+  return translation[fieldKey].options[fieldValue].label;
+};
+
+const getPersonalisation = (recipientType, req) => {
+  const basePersonalisation = {
+    is_tenant: req.sessionModel.get('isCurrentTenant') ?? '',
+    is_prospective_tenant: !req.sessionModel.get('isCurrentTenant') ?? '',
+    existing_occupier: getLabel('person-live-in', req.sessionModel.get('person-live-in')),
+    is_existing_occupier: getLabel('person-live-in', req.sessionModel.get('person-live-in')) === 'Yes' ? 'yes' : 'no',
```

Comment:
no need to fetch label here, should be replaced:

`is_existing_occupier: (req.sessionModel.get('person-live-in') === 'yes' ? 'yes' : 'no',`

NOTE: left check on value here deliberately, in case the value for the field is replaced in the future.

but could be removed if chance is low
`is_existing_occupier: req.sessionModel.get('person-live-in'),`